pytest_selenium/pytest_selenium.py

```diff
@@ -265,6 +262,16 @@ def _gather_logs(item, report, driver, summary, extra):
                 format_log(log), '%s Log' % name.title()))
 
 
+def _gather_driver_logs(item, summary, extra):
+    pytest_html = item.config.pluginmanager.getplugin('html')
+    if hasattr(item.config, '_driver_log') and \
+       os.path.exists(item.config._driver_log):
```

Comment:
This is interesting. Was `item.config._driver_log` not `None`? I would actually expect this to fail with the details from your initial comment, as a `None` value here would pass the first condition and then fail the second.

Reply:
I was getting an attribute error.

Reply:
We explicitly set the `_driver_log` attribute here: https://github.com/pytest-dev/pytest-selenium/blob/a90604245a1902f320e2caecbf2187e20355bac1/pytest_selenium/pytest_selenium.py#L104

This means it should be save to check the value rather than the presence. Can you provide the traceback for the error you were seeing?